script.js

```diff
@@ -1,25 +1,29 @@
 "use strict"
 
-const small_film_set = [
-    { id: 1, title: "The Shawshank Redemption", year: 1994, votes: 678790, rating: 9.2, rank: 1, category: "Thriller" },
-    { id: 2, title: "The Godfather", year: 1972, votes: 511495, rating: 9.2, rank: 2, category: "Crime" },
-    { id: 3, title: "The Godfather: Part II", year: 1974, votes: 319352, rating: 9.0, rank: 3, category: "Crime" },
-    { id: 4, title: "The Good, the Bad and the Ugly", year: 1966, votes: 213030, rating: 8.9, rank: 4, category: "Western" },
-    { id: 5, title: "Pulp fiction", year: 1994, votes: 533848, rating: 8.9, rank: 5, category: "Crime" },
-    { id: 6, title: "12 Angry Men", year: 1957, votes: 164558, rating: 8.9, rank: 6, category: "Western" }
-]
-
 function saveItem() {
-    const validationResult = $$("form").validate();
-    if (validationResult) {
-        const item_data = $$("form").getValues();
-        $$("table").add(item_data);
-        $$("form").clear();
-        webix.message({
-            text: `Validation is successful`,
-            type: "success",
-            expire: 1000
-        });
+    const form = $$("form");
+    const table = $$("table");
+
+    if (form.validate()) {
+        const item_data = form.getValues();
+        if (item_data.id) {
+            table.updateItem(item_data.id, item_data);
+            form.clear();
```

Comment:
form.clear is a common action, it's better to place it after if..else block

Reply:
Thank you, moved it out from if...else block.